### PR TITLE
Updated stable discovery version to 8.0

### PR DESCRIPTION
**Pull request by lzap**
I am shooting in the dark here.

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_discovery.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     branch:
       - develop:
           foreman_branch: develop
-      - 4.0-stable:
-          foreman_branch: 1.9-stable
+      - 7.0-stable:
+          foreman_branch: 7.0-stable
```
Comment (mmoll) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_discovery.yaml]: that should be 1.13(or 1.14?)-stable, I guess

Comment (lzap) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_discovery.yaml]: Ah yeah fixing.

**Comment by lzap** [comment]
Updated to 8.0 as I am releasing this today.

**Comment by domcleal** [comment]
Merged, a new 8.0-stable testing job should be created within an hour or so.

**Pull request merged**